### DISCUSSION TITLE
If I use GitHub Copilot's GitHub Copilot Labs, will my data be shared that I don't want?

### DISCUSSION THREAD

**Discussion by wearekrafton** [community/community#52767] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I'd like to be able to see if there's any chance of a data leak or something.
[GitHub Next Pre-Release License Terms](https://github.com/githubnext/githubnext/blob/main/TERMS_AND_CONDITIONS.md)

Best Regards,
Yongwon BAEK.

**Comment by Akash1134** [comment, marked as answer]
Hi @wearekrafton ,

Copilot Labs is an experimental product(technical preview) and has a different scope of data that is collected which is purely to provide services and improve our products and services. GitHub will use data for analytics and measurement to understand how the product functions and can be improved further. Your data is used for training the experimental product, if you do not wish to share it please use [Copilot](https://github.com/features/copilot) 

You may checkout the section [Data](https://github.com/githubnext/githubnext/blob/main/TERMS_AND_CONDITIONS.md#:~:text=any%20third%20party.-,DATA,-.) to know more about the information collected and how you can control it.
